app/frontend/helpers/formatDate.js

```diff
@@ -0,0 +1,10 @@
+const formatDate = (date) => {
+  const timeArray = new Date(date * 1000).toLocaleString().split(',')
+  const numberOfSpaces = 10 - timeArray[0].length
+  const numberOfSpaces2 = 12 - timeArray[1].length
```

Comment:
this may fail for locales different that en-US, let's say the slovak one

Reply:
other solution could be to force en-US locale but I would prefer to use browser's locale for optimal UX

Reply:
Yup, totally agree, and it even looks better.